include/util/name_table.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef OSRM_UTIL_NAME_TABLE_HPP
+#define OSRM_UTIL_NAME_TABLE_HPP
+
+#include "util/shared_memory_vector_wrapper.hpp"
+#include "util/range_table.hpp"
+
+#include <string>
+
+namespace osrm
+{
+namespace util
+{
+class NameTable
```

Comment:
what is a name table good for, what's its purpose? skimming the class says nothing about it. I assume for street names?